test/unit/cookies.test.js

```diff
@@ -41,6 +42,18 @@ describe('cookies', () => {
     expect(h.state).not.toHaveBeenCalled()
   })
 
+  test('validSession returns true if the session is valid ', () => {
+    request.state.cookies_policy = { confirmed: true, essential: false, analytics: true }
```

Comment:
Should this be `request.state.session = true`?

Reply:
done